### PR TITLE
Add google login

### PR DESCRIPTION
**Pull request by SaraVieira**

### CODE REVIEW

File: vercel.json [hunk 1]
```diff
@@ -1,7 +0,0 @@
-{
-  "build": {
-    "env": {
-      "ENABLE_FILE_SYSTEM_API": "1"
-    }
-  }
-}
```
Comment (SaraVieira) [inline vercel.json:7]: Got this message in vercel
CleanShot 2022-07-02 at 19 53 40@2x

**Comment by SaraVieira** [comment]
Working on: https://webstudio-designer-34nl7vf4d-webstudio-is.vercel.app/

File: app/routes/login/index.tsx [hunk 2]
```diff
@@ -15,7 +15,13 @@ export const loader: LoaderFunction = async ({ request }) => {
     return redirect(config.dashboardPath);
   }
 
-  return { devLogin: process.env.DEV_LOGIN === "true", env };
+  return {
+    devLogin: process.env.DEV_LOGIN === "true",
```
Comment (kof) [inline app/routes/login/index.tsx:19]: can we call the booleans with is/has prefix please? isDevLogin, isGithubEnabled etc?

File: app/auth/login.tsx [hunk 3]
```diff
@@ -52,15 +52,15 @@ export const Login = ({ errorMessage }: { errorMessage: string }) => {
...
+              <Button type="submit" disabled={!loaderData.githubEnabled}>
                 <Flex gap="1">
                   <GithubIcon width="16" />
                   Login with GitHub
                 </Flex>
               </Button>
             </Form>
             <Form action="/auth/google" method="post">
-              <Button type="submit" disabled>
+              <Button type="submit" disabled={!loaderData.googleEnabled}>
```
Comment (kof) [inline app/auth/login.tsx:63]: `!l` and`!i` is hard to read, please use 
```suggestion
              <Button type="submit" disabled={loaderData.isGoogleEnabled === false}>
```

Comment (SaraVieira) [reply app/auth/login.tsx]: Fixed

**Review by kof — APPROVED** [review]
🎉

**Pull request merged**